src/pages/EditUserPage.ts

```diff
@@ -0,0 +1,94 @@
+import { expect } from '@/fixtures/fixtures';
+import BasePageModel from '@/pages/BasePageModel';
+
+class EditUserPage extends BasePageModel {
+  async isLoaded() {
+    await expect(this.page.locator('title')).toBeVisible();
+  }
+
+  get userTitle() {
+    return this.page.locator('.summary .title');
+  }
+
+  get actionButton() {
+    return this.page.locator('button.action-btn');
+  }
+
+  get deleteUserButton() {
+    return this.page.getByRole('menuitem').filter({ hasText: 'Delete User' });
+  }
+
+  get activateUser() {
+    return this.page.getByRole('checkbox').nth(0);
```

Comment:
create a selector for user details section
```ts
  get userDetailsSection() {
    return this.page.getByRole('region', { name: 'User Details' });
  }
```
and select this checkbox from that section
```ts
    return this.userDetailsSection.gegetByRole('checkbox').nth(0);
```